OSpackages.sh

```diff
@@ -125,10 +125,10 @@ function GetDefaultRepos {
    fi
 
    case $( rpm -qf /etc/os-release --qf '%{name}' ) in
-      centos-release)
+      centos-stream-release)
```

Comment:
Let's add a new case statement instead. That way it continues to work for the images where this *is* `centos-release`.

Reply:
updated to add the appropriate match for "centos-linux-release"